packages/client/components/developers/DeveloperList.tsx

```diff
@@ -1,8 +1,20 @@
 import { Box, Grid } from '@chakra-ui/react';
-
+import { useState, useEffect } from 'react';
+import { supabase } from '@/common/supabase';
+import { User } from '@/types/user';
 import DeveloperItem from './DeveloperItem';
 
 export default function DeveloperList() {
+  const [devs, setDevs] = useState<User[]>([]);
+  useEffect(() => {
+    fetchUsers();
+  }, []);
+
+  const fetchUsers = async () => {
+    const { data, error } = await supabase.from('users').select();
+    setDevs(data || []);
+  };
```

Comment:
We might wanna put a where statement in this query, so to ensure it's only developers we are showing in the list.